### ISSUE TITLE
Getting started

### ISSUE THREAD

**Issue by smjanke** [NQCD/NonadiabaticMolecularDynamics.jl#107]
https://github.com/NQCD/NonadiabaticMolecularDynamics.jl/blob/6039a6c2e0c9be83868f8f0fc9a04d6fd7e978f6/docs/src/getting_started.md#L176

This is unclear. How do you access this table and this u?

**Comment by jamesgardner1421** [comment]
Yeah let's add an extra box before the plot that shows the structure of the table and how to access each column
